test/controllers/recipes_controller_test.rb

```diff
@@ -0,0 +1,29 @@
+require "test_helper"
+
+describe RecipesController do
+
+  it "can get the root path" do
+    VCR.use_cassette('recipes') do
+      get root_path
+      must_respond_with :ok
+    end
+
+  end
+
+it "can get the index path" do
+  VCR.use_cassette('recipes') do
+    get recipes_path
+    must_respond_with :ok
+  end
```

Comment:
There are some interesting cases you're not covering here:

- What happens if no search term is provided?
- What if the search returns no results?

It might also be worthwhile to add some tests around the paging parameters:

- What happens if they're absent?
- Do you get different results when they change?
- What if you send a bogus value, like a negative page number?